.stylelintrc

```diff
@@ -13,3 +13,4 @@ rules:
   rule-empty-line-before: null
   selector-pseudo-element-colon-notation: null
   shorthand-property-no-redundant-values: true
+  value-keyword-case: null
```

Comment:
Had to disable this because stylelint started complaining about font case name in css vars, but was apparently fine with with it in the less vars. We could go the route of lowercasing all font names but I have a slight preference for keeping the font name case here.

Reply:
Actually, found a workaround by always quoting specific font names.